boards/arm/nucleo_f411re/pinmux.c

```diff
@@ -22,6 +22,14 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA2, STM32F4_PINMUX_FUNC_PA2_USART2_TX},
 	{STM32_PIN_PA3, STM32F4_PINMUX_FUNC_PA3_USART2_RX},
 #endif	/* CONFIG_UART_STM32_PORT_2 */
+#ifdef CONFIG_I2C_1
+	{STM32_PIN_PB8, STM32F4_PINMUX_FUNC_PB8_I2C1_SCL},
+	{STM32_PIN_PB9, STM32F4_PINMUX_FUNC_PB9_I2C1_SDA},
+#endif /* CONFIG_I2C_1 */
+#ifdef CONFIG_I2C_2
+	{STM32_PIN_PB10, STM32F4_PINMUX_FUNC_PB10_I2C2_SCL},
+	{STM32_PIN_PB3, STM32F4_PINMUX_FUNC_PB3_I2C2_SDA},
```

Comment:
Is that PB2 or PB3? 
You mention PB2 in nucleof411re.rst

Reply:
You are quite right. Changed to PB3 in nucleof411re.rst